DotNet/Report/appsettings.json

```diff
@@ -21,6 +21,19 @@
       }
     }
   },
+  "Quartz": {
```

Comment:
Update Report service documentation to include all of these configurable options and a description (and mention of default values).